main/player.py

```diff
@@ -3,5 +3,42 @@
 
 
 class Player(Character):
-    def __init__(self):
-        super().__init__()
+    def __init__(self, x, y, name, strength, defense, health_points):
+        super().__init__(x, y, name, strength, defense, health_points)
+        self.backpack = []
+        self.backpack_size = 4
+        self.level = 1
+        self.experience = 0
+        self.experience_needed = 100
+        self.attribiute_points = 10
+        self.money = 0
+
+    def add_to_backpack(self, item):
+        if len(self.backpack) < self.backpack_size:
+            self.backpack.append(item)
+        else:
+            print("Backpack is full")
+
+    def remove_from_backpack(self, item):
+        is_item = False
+        for i in self.backpack:
+            if i == item:
+                self.backpack.remove(item)
+                is_item = True
+        print(f"{item} removed from backpack")
+        if not is_item:
+            print(f"{item} not found in backpack")
```

Comment:
If you need to check if something is in list, tuple, dict etc. it is better to use `if <something> in <some_list`, instead of for loop iteration:
```suggestion
    def remove_from_backpack(self, item):
        if item in self.backpack:
            self.backpack.remove(item)
            print(f"{item} removed from backpack")
        else:
            print(f"{item} not found in backpack")
```